rmgpy/molecule/group.py

```diff
@@ -433,6 +453,34 @@ def copy(self):
         """
         return GroupBond(self.vertex1, self.vertex2, self.order[:])
 
+    def isSingle(self):
+        """
+        Return ``True`` if the bond represents a single bond or ``False`` if
+        not.
+        """
+        return self.order[0] == 'S' and len(self.order) == 1
```

Comment:
I'll add a comment a clarifying comment, but I think that is the behavior we want.